docs/Guide.md

```diff
@@ -50,7 +50,7 @@ class MyWeb extends Component {
 }
 ```
 
-### Add support for File Upload
+### Add support for File Upload / Download
```

Comment:
I recommend to put the `Download` section separately, because users might get the impression that they also need to add the iOS settings for the Download, as well.

Reply:
@andreipfeiffer How does that look? iOS only needs the permission added for download.